app/models/claim.rb

```diff
@@ -19,4 +19,11 @@ def self.fetch_all(headers)
       end
     end.flatten
   end
+
+  def self.upload_document(headers, file_name, file_body, claim_id, tracked_item_id)
```

Comment:
Do we know if EVSS has any authorization rules? My preference would be to keep our app as simple as possible and just pass through all requests and let EVSS return 401/404 in any cases where the user does not have permission